src/test/resources/features/refapp-2.x/05-location/locations.feature

```diff
@@ -1 +1,28 @@
-Feature: Location Management
\ No newline at end of file
+Feature: Location Management
+
+  Background:
+
+
```

Comment:
what is the background of the user stories?

Reply:
Did you see this comment